pom.xml

```diff
@@ -159,14 +159,6 @@
                     <execution>
                         <id>checkstyle-validation</id>
                         <phase>validate</phase>
-                        <configuration>
-                            <configLocation>src/checkstyle/checkstyle.xml</configLocation>
-                            <headerLocation>src/checkstyle/checkstyle-header.txt</headerLocation>
-                            <encoding>UTF-8</encoding>
-                            <consoleOutput>true</consoleOutput>
-                            <failOnViolation>true</failOnViolation>
-                            <includeTestSourceDirectory>true</includeTestSourceDirectory>
-                        </configuration>
```

Comment:
Is it okay to remove this? Won't checkstyle stop working without it?

Reply:
Isn't this configuration duplicated?

Reply:
I am not sure. They're in different blocks. We should check that before merging this.